packages/core/__tests__/core.test.ts

```diff
@@ -387,3 +388,20 @@ function verifyFileCommand(command: string, expectedContents: string): void {
     fs.unlinkSync(filePath)
   }
 }
+
+function getTokenEndPoint() {
+  return 'https://vstoken.actions.githubusercontent.com/.well-known/openid-configuration'
+}
+
+describe('oidc-client-tests', () => {
```

Comment:
what are these tests for? I would expect some L0 tests around `IOidcClient`

Reply:
added more L0 tests